src/components/Dropdown.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+class Dropdown extends React.Component {
+
+
+    render() {
+        const disabled = this.props.disabled || this.props.items.length === 0;
+        const dropdownItems = this.props.items.map((item) => 
+            <button className="dropdown-item" disabled={item.disabled} key={item.text} onClick={item.onClick}>{item.text}</button>
+        );
+        const active = this.props.active ? "active" : "";
```

Comment:
I would combine these rather than checking this.props.active? twice
```
if (this.props.active) {
    const active = "active";
    const label = this.props.activeLabel;
} else {
...
```